fuel-vm/src/tests/blockchain.rs

```diff
@@ -453,78 +447,19 @@ fn ldc_reason_helper(
 
     client.deploy(tx_create_target);
 
-    // test ssp != sp for LDC
-    let mut load_contract: Vec<Instruction>;
-
-    let mut tx_deploy_loader;
-
-    if !should_patch_jump {
-        load_contract = cmd;
-
-        tx_deploy_loader =
-            TransactionBuilder::script(load_contract.into_iter().collect(), vec![])
-                .gas_price(gas_price)
-                .gas_limit(gas_limit)
-                .maturity(maturity)
-                .add_random_fee_input()
-                .finalize()
-                .into_checked(height, &consensus_params)
-                .expect("failed to check tx");
-    } else {
-        let reg_a = 0x20;
-        let count = ContractId::LEN as Immediate12;
-
-        load_contract = vec![
-            op::xor(reg_a, reg_a, reg_a), // r[a] := 0
-            op::ori(reg_a, reg_a, count), // r[a] := r[a] | ContractId::LEN
-            op::aloc(reg_a),              // Reserve space for contract id in the heap
-        ];
-
-        // Generate code for pushing contract id to heap
-        for (i, byte) in contract_id.as_ref().iter().enumerate() {
-            let index = i as Immediate12;
-            let value = *byte as Immediate12;
-            load_contract.extend([
-                op::xor(reg_a, reg_a, reg_a),    // r[a] := 0
-                op::ori(reg_a, reg_a, value),    // r[a] := r[a] | value
-                op::sb(RegId::HP, reg_a, index), // m[$hp+index] := r[a] (=value)
-            ]);
-        }
```

Comment:
I'm not sure I understand what any of this was doing, or why we no longer need it. Can you provide any context here?

Reply:
This code was all in the `true` branch of the if/else. But we only ever used `false` in any of our tests. It's dead code.

Reply:
Gotcha 👍